mmv1/products/networkconnectivity/RegionalEndpoint.yaml

```diff
@@ -0,0 +1,127 @@
+# Copyright 2024 Google Inc.
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# # you may not use this file except in compliance with the License.
+# # You may obtain a copy of the License at
+# #
+# #     http://www.apache.org/licenses/LICENSE-2.0
+# #
+# # Unless required by applicable law or agreed to in writing, software
+# # distributed under the License is distributed on an "AS IS" BASIS,
+# # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# # See the License for the specific language governing permissions and
+# # limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'RegionalEndpoint'
+description: |
+  Regional Private Service Connect (PSC) endpoint resource.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Access regional Google APIs through endpoints': 'https://cloud.google.com/vpc/docs/access-regional-google-apis-endpoints'
+  api: 'https://cloud.google.com/network-connectivity/docs/reference/networkconnectivity/rest/v1/projects.locations.regionalEndpoints'
+base_url: 'projects/{{project}}/locations/{{location}}/regionalEndpoints'
+self_link: 'projects/{{project}}/locations/{{location}}/regionalEndpoints/{{name}}'
+immutable: true
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 20
+  delete_minutes: 20
```

Comment:
The timeouts is not needed. The default timeouts is 20 min

Reply:
Done.